src/caret_analyze/architecture/architecture.py

```diff
@@ -399,6 +453,19 @@ def remove_publisher_callback(self, node_name: str,
 
     def remove_variable_passing(self, node_name: str,
                                 callback_name_write: str, callback_name_read: str) -> None:
+        """
+        Remove variable_passing in "node_name" node.
+
+        Parameters
+        ----------
+        node_name : str
+            name of target node
+        callback_name_write : str
+            name of write callback in variable_passing to remove
+        callback_name_read : str
+            name of read callback in variable_passing to remove
```

Comment:
```suggestion
            name of write callback to be removed from variable_passing
        callback_name_read : str
            name of read callback to be removed from variable_passing
```

Reply:
Fix it in [6cf0285](https://github.com/tier4/CARET_analyze/pull/300/commits/6cf0285fa738613d5ed20fe836eed60ce682205c).